### ISSUE TITLE
Feature Request: Resource for managing AWS Glue Connections

### ISSUE THREAD

**Issue by bflad** [terraform-providers/terraform-provider-aws#3874]
Split feature request from #1416. For issues not immediately being worked on, please use :+1: upvotes on this original issue comment to help guage community interest.

### Terraform Version

```
terraform 0.10+
terraform-provider-aws (new feature)
```

### Affected Resource(s)

- `aws_glue_connection`

### Terraform Configuration Files

```hcl
# New Resource - not implemented
resource "aws_glue_connection" "example" {
  # ...
}
```

### Expected Behavior

Create, read, update, and delete support for AWS Glue connections.

### Actual Behavior

Resource not implemented.

### References

* https://docs.aws.amazon.com/glue/latest/dg/aws-glue-api-catalog-connections.html
* https://docs.aws.amazon.com/sdk-for-go/api/service/glue/#Glue.CreateConnection

**Comment by bflad** [comment]
I will be taking a look at implementing this starting next week.

**Comment by bflad** [comment]
PR submitted: #4016

**Comment by bflad** [comment]
The new resource has been merged into master and will release with v1.15.0 of the AWS provider, likely mid next week. 🎉

**Comment by bflad** [comment]
This has been released in [version 1.15.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1150-april-18-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
